specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -284,6 +284,52 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/hanaInstances/{hanaInstanceName}/monitoring": {
+      "post": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "HanaInstances_Monitoring",
+        "x-ms-long-running-operation": true,
+        "description": "The operation to monitor a SAP HANA instance.",
+        "x-ms-examples": {
+          "Monitor a HANA instance": {
+            "$ref": "./examples/HanaInstances_Monitoring.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HanaInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitoringParameter"
```

Comment:
I haven't added non-path parameters to the Swagger spec before; is this how they're meant to be specified?

Reply:
fixed

Reply:
I believe so because that is also the way that it is done for tags